R/logo.R

```diff
@@ -49,7 +49,13 @@ use_logo <- function(img, geometry = "240x278", retina = TRUE) {
   }
 
   ui_todo("Add logo to your README with the following html:")
-  ui_code_block("# {pkg} <img src=\"{ui_path(logo_path)}\" align=\"right\" height=\"{height}\" />")
+
+  pd_link <- pkgdown_link()
+  if (!is.null(pd_link)) {
+    ui_code_block("# {pkg} <a href={ui_path(pd_link)}><img src={ui_path(logo_path)} align=\"right\" height=\"{height}\" /></a>")
+  } else {
+    ui_code_block("# {pkg} <img src={ui_path(logo_path)} align=\"right\" height=\"{height}\" />")
```

Comment:
I removed the double quotes here, because `ui_path()` single quotes it's contents, so you had double quotes around single quotes which seemed redundant.

Reply:
Good catch, thanks!